python/lsst/ip/isr/isrTask.py

```diff
@@ -1382,6 +1382,18 @@ def run(self, ccdExposure, camera=None, bias=None, linearizer=None,
                         self.metadata.set(f"ISR OSCAN {amp.getName()} STDEV", qaStdev)
                         self.log.debug("  Overscan stats for amplifer %s: %f +/- %f",
                                        amp.getName(), qaMedian, qaStdev)
+
+                        # Residuals after overscan correction
+                        qaStatsAfter = afwMath.makeStatistics(overscanResults.overscanImage,
+                                                              afwMath.MEDIAN | afwMath.STDEVCLIP)
+                        qaMedianAfter = qaStatsAfter.getValue(afwMath.MEDIAN)
+                        qaStdevAfter = qaStatsAfter.getValue(afwMath.STDEVCLIP)
+
+                        self.metadata.set(f"ISR OSCAN {amp.getName()} MEDIAN AFTER CORRECTION", qaMedianAfter)
```

Comment:
Can you change these to  {{RESIDUAL MEDIAN}} and {{RESIDUAL STDEV}}?  I think that might be easier to understand.  Changing the ones above to {{FIT MEDIAN}} and {{FIT STDEV}} might help as well.